fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/loader/util/COSObject.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * (C) Copyright IBM Corp. 2021, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.term.graph.loader.util;
+
+import java.io.InputStreamReader;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.cloud.objectstorage.ClientConfiguration;
+import com.ibm.cloud.objectstorage.SDKGlobalConfiguration;
+import com.ibm.cloud.objectstorage.SdkClientException;
+import com.ibm.cloud.objectstorage.auth.AWSCredentials;
+import com.ibm.cloud.objectstorage.auth.AWSStaticCredentialsProvider;
+import com.ibm.cloud.objectstorage.client.builder.AwsClientBuilder;
+import com.ibm.cloud.objectstorage.oauth.BasicIBMOAuthCredentials;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3ClientBuilder;
+import com.ibm.cloud.objectstorage.services.s3.model.GetObjectRequest;
+import com.ibm.cloud.objectstorage.services.s3.model.S3Object;
+
+/*
+ * This is a helper class capable of retrieving cloud object store (COS) objects. It relies on environment variables for configuration.
+ */
+public class COSObject {
+    private static String DEFAULT_COS_BUCKET_LOCATION = "us"; // eg "us"
+
+    private static String DEFAULT_COS_ENDPOINT = "https://s3.us-east.cloud-object-storage.appdomain.cloud";
+
+    private static final Logger LOG = Logger.getLogger(COSObject.class.getName());
+
+    /**
+     * Create client connection
+     */
+    private static AmazonS3 createClient(String api_key, String service_instance_id, String endpoint_url,
+            String location) {
+        AWSCredentials credentials;
+        credentials = new BasicIBMOAuthCredentials(api_key, service_instance_id);
+
+        ClientConfiguration clientConfig = new ClientConfiguration().withRequestTimeout(5000);
+        clientConfig.setUseTcpKeepAlive(true);
+
+        AmazonS3 cosClient = AmazonS3ClientBuilder.standard()
+                .withCredentials(new AWSStaticCredentialsProvider(credentials))
+                .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(endpoint_url, location))
+                .withPathStyleAccessEnabled(true).withClientConfiguration(clientConfig).build();
+        return cosClient;
+    }
+
+    private static final String getCOSAPIKeyId() {
+        return System.getenv("COS_API_KEY_ID"); // e.g. "KrgAwCqAIxyRI855y0oVaeQopHokkqFNiReyQuwHQnDY"
```

Comment:
These should all be prefixed. 
Let's not use COS_API_KEY_ID 
Let's take out that e.g. as well

Reply:
I've added "TERM_" to all environment variables in the loader logic. and the examples I have (1) are not real values - I changed them enough to make it safe to deliver, and (2) I've added  "(not a real xxx)" to the end of each to make sure people realize they're safe.  I like having them there to make sure I know the appropriate/expected format for values of that field.  I can remove them if you really think they're problematic.